src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -272,7 +272,7 @@ function ChartDeploymentHistory({
         return (
             <>
                 {deploymentHistoryArr.map((deployment, index) => {
-                    const helmDeploymentStatus = deployment?.status.toLowerCase() || 'succeeded'
+                    const helmDeploymentStatus: string = deployment?.status ? deployment?.status.toLowerCase() : 'succeeded'
```

Comment:
this should be written as `const helmDeploymentStatus: string = deployment?.status ? deployment.status.toLowerCase() : 'succeeded'`

Reply:
Updated 👍